tmt/steps/prepare/__init__.py

```diff
@@ -88,9 +93,14 @@ def go(self):
 
         # Prepare guests (including workdir sync)
         for guest in self.plan.provision.guests():
+            # Make sure rsync is installed, push the workdir
+            if rsync_required:
+                self.debug('Ensure that rsync is installed on the guest.')
+                guest.execute('rpm -q rsync || yum install -y rsync')
```

Comment:
What about `dnf`?

Reply:
I tested with Fedora images, `yum` seems to be always present and we need this working for older distros as well.

Reply:
`yum`  is also as backward compatibility there so `yum` should be good enough. I've also though about that point